scripts/calculateRewards/proofOfImpact.ts

```diff
@@ -36,20 +36,17 @@ export function calculateRewardsProofOfImpact({
   )
   const totalRewardsForPeriod = timeDiff.times(rewardsPerMillisecond)
 
-  const referrerKpis = kpiData
-    // filter out rows with no revenue, which is possible for users who were referred between the reward period end date and the time of the reward distribution
-    .filter((row) => BigInt(row.revenue) > 0)
```

Comment:
i realised this filter needs to go on all rewards, because it is always possible that the revenue calculation for a referred user is 0 (if a builder doesn't not attach the referral to a KPI transaction). this is only needed so that we don't post 0 rewards, so i moved it to the createSafeTransactionBatch function instead